### PR TITLE
TW-630 Update Azure AD SCIM docs

### PR DESCRIPTION
**Pull request by claireriley**

### CODE REVIEW

**Review by MatthewPinsker-postman — APPROVED** [review]
Looks good to me!

**Review by petewitcher — APPROVED** [review]
LGTM

File: src/pages/docs/administration/scim-provisioning/configuring-scim-with-azure-ad.md [hunk 1]
```diff
@@ -55,10 +55,17 @@ Next, you will configure the Azure AD integration.
 
 After you set up SCIM in Azure AD, you can configure the integration with Postman:
 
-1. In the Azure AD **Mappings** section, enable **Provision Azure Active Directory Users**. This is the set of attribute mappings for user objects. Select it to review the attributes that are synchronized from Azure Active Directory to Postman. **Provision Azure Active Directory Groups** needs to be turned off.
+1. In the Azure AD **Mappings** section, enable **Provision Azure Active Directory Users**. This is the set of attribute mappings for user objects. Select it to review the attributes that are synchronized from Azure Active Directory to Postman.
 
     > The attributes you select as **Matching** properties are used to match the users in Postman for update operations.
 
+1. (Optional) Enable **Provision Azure Active Directory Groups** by selecting **Yes**.
+1. (Optional) Under **Target Object Actions**, enable **Create**, **Update**, and **Delete**.
```
Comment (HeatherStenson) [inline src/pages/docs/administration/scim-provisioning/configuring-scim-with-azure-ad.md:63]:
```suggestion
1. (Optional) Turn on **Provision Azure Active Directory Groups** by selecting **Yes**.
1. (Optional) Under **Target Object Actions**, turn on **Create**, **Update**, and **Delete**.
```

This is sort of the flip side of "[disable/disabled](https://postmanlabs.atlassian.net/wiki/spaces/TW/pages/3828744395/Inclusive+Language)", but we don't have a specific rule about it yet. (Microsoft says [no](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/enable-enables), fwiw, and I'll add this to the discussion backlog for us. Feel free to ignore this suggestion in the meantime if you want!)

**Review by HeatherStenson — APPROVED** [review]
Looks great! 🍊

File: src/pages/docs/administration/scim-provisioning/configuring-scim-with-azure-ad.md (hunk 1 above)
Comment (johnoffenhartzPM) [reply src/pages/docs/administration/scim-provisioning/configuring-scim-with-azure-ad.md:63]: Looks good. Also I tend to favor "enable" as that's the more "technically correct" language (yes, this is pedantic)

**Pull request merged**